protostar/cairo/short_string.py

```diff
@@ -1,8 +1,24 @@
 CairoShortString = int
 
+LAST_ASCII_CHAR_DEC_VAL = 127
+
 
 def short_string_to_str(value: CairoShortString) -> str:
     """
     Reverse of starkware.cairo.lang.compiler.test_utils.short_string_to_felt
     """
-    return value.to_bytes(length=32, byteorder="big").decode("ascii").lstrip("\x00")
+    return _to_bytes(value).decode("ascii").lstrip("\x00")
+
+
+def is_short_string(value: int) -> bool:
+    if value > int.from_bytes(b"\xff" * 31, "big"):
+        return False
```

Comment:
Isn't this check redundant if we are validating every character to be in ASCII range below?

Reply:
No, it is not. There is a possibility that passed fetl252 will be too large to fit in 31 bytes (248 bits). In this case we can be sure it was not a short string literal and just refrain from decoding it, as it doesn't make any sense